### ISSUE TITLE
Run with specific version

### ISSUE THREAD

**Issue by sefaphlvn** [tetratelabs/func-e#313]
**Is your feature request related to a problem? Please describe.**
I am running multiple services and I would like to be able to run each of them in a different version. Currently, all services can work with a single version.

**Describe the solution you'd like**
I was able to run getenvoy as follows
for version 1.18.3: getenvoy run standard:1.18.3/linux-glibc -- --config-path bootstrap1.json
for version 1.17.1: getenvoy run standard:1.17.1/linux-glibc -- --config-path bootstrap2.json

**Describe alternatives you've considered**
It would be nice to be able to specify version and run like in getenvoy example:
func-e run 1.18.2 -c bootstrap1.json

**Comment by codefromthecrypt** [comment]
TL;DR; is this is supported via env. ex. `ENVOY_VERSION=1.18.3 func-e run -c ...`

This isn't done as a flag as doing so would mix with what envoy interprets. You can also choose to add a `.envoy-version` file to accomplish the same, if for example this is a project setup.

---
sorry I guess our site isn't publishing details per command, though the help statement will:
https://github.com/tetratelabs/func-e/blob/master/USAGE.md

```bash
$ func-e help run
NAME:
   func-e run - Run Envoy with the given [arguments...] until interrupted

USAGE:
   func-e run [arguments...]

DESCRIPTION:
   To run Envoy, execute `func-e run -c your_envoy_config.yaml`.
   
   The first version in the below is run, controllable by the "use" command:
   ```
   $ENVOY_VERSION, $PWD/.envoy-version, $FUNC_E_HOME/version
   ```
   The version to use is downloaded and installed, if necessary.
   
   Envoy interprets the '[arguments...]' and runs in the current working
   directory (aka $CWD) until func-e is interrupted (ex Ctrl+C, Ctrl+Break).
   
   Envoy's process ID and console output write to "envoy.pid", stdout.log" and
   "stderr.log" in the run directory (`$FUNC_E_HOME/runs/$epochtime`).
   When interrupted, shutdown hooks write files including network and process
   state. On exit, these archive into `$FUNC_E_HOME/runs/$epochtime.tar.gz`
```

**Comment by codefromthecrypt** [comment]
I will refactor the help text so that the ENV option is more clear also and ping this issue when that happens

**Comment by sefaphlvn** [comment]
Hmm if I add to environment variable to systemd file my problem is solve right? func-e command in sh file

```Environment="ENVOY_VERSION=1.18.3"```

```
[Unit]
Description={{.Domain}}
Requires=network-online.target
After=network-online.target

[Service]
Type=simple
Environment="ENVOY_VERSION=1.18.3"
User=envoyuser
Group=envoyuser
ExecStart=/usr/envoy_hotrestarters/{{.Domain}}_hotrestart.py {{.DomainPath}}{{.Domain}}/{{.Domain}}.sh
ExecReload=/bin/kill -HUP $MAINPID
ExecStop=/bin/kill -TERM $MAINPID
Restart=on-failure
RestartSec=30
StandardError=syslog
#!/bin/bash
SyslogIdentifier={{.Domain}}
LimitNOFILE=1048576

[Install]
WantedBy=multi-user.target
```

**Comment by codefromthecrypt** [comment]
I'm not familiar with the syntax enough to proofread, but definitely if you are using func-e recent version, that ENV will work.

**Comment by sefaphlvn** [comment]
it's working thank you.
